websites/R/Roblox/presence.ts

```diff
@@ -33,14 +36,18 @@ presence.on("UpdateData", async () => {
   ) {
     profileName = <HTMLHeadingElement>(
       document.querySelector(
-        "div.profile-header-top > div.header-caption > div.header-title > h2"
+        "div.profile-header-top > div.header-caption div.header-title > h2"
       )
     );
 
     profileTabs = <HTMLAnchorElement>(
       document.querySelector("#horizontal-tabs li.rbx-tab.active a")
     );
 
+    profileAvatar = <HTMLImageElement>(
+      document.querySelector(".avatar-headshot-lg thumbnail-2d img[image-load]")
+    );
```

Comment:
Do this everywhere else too
```suggestion
    profileAvatar = document.querySelector<HTMLImageElement>(".avatar-headshot-lg thumbnail-2d img[image-load]");
```